sw/device/silicon_creator/lib/sigverify/spx_key.h

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_SPX_KEY_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_SPX_KEY_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  /**
+   * Length of an SPX public key in bits.
+   */
+  kSigverifySpxKeyNumBits = 128,
```

Comment:
nit: 
```suggestion
  kSigverifySpxKeyNumBits = 256,
```

Otherwise this looks like a good direction to me :slightly_smiling_face:

Reply:
Fixed, thanks!